server/tests/security/test_command_safety_runtime.py

```
@@ -0,0 +1,344 @@
+    def test_long_input_still_returns_16_chars(self):
+        fp = _fingerprint("a" * 100_000)
+        assert len(fp) == 16
+        assert set(fp) <= _HEX_ALPHABET
+
+
+# ---------------------------------------------------------------------------
+# evaluate_command pipeline -- signature matcher → LLM judge
+# ---------------------------------------------------------------------------
+
+
+def _silence_audit(monkeypatch):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_silence_audit`

Add return type annotation: `None`

(ANN202)